porcupine/plugins/filemanager.py

```diff
@@ -122,6 +125,24 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    global copy_path
+    new_file_path = os.path.join(new_path, os.path.basename(copy_path))
```

Comment:
Porcupine uses pathlib, which makes things like this a lot easier. You can probably find some guides for how to use it with google. For example, this line is best written like this:

```suggestion
    new_file_path = new_path / copy_path.name
```

Let me know if you need help with rewriting the code to use pathlib.

Reply:
Didn't know about it. I'll check it!